### PR TITLE
[misc][xwalk-4132] Integrate M43 new feature samples

### PR DESCRIPTION
**Pull request by chunywang**
Integrate 2 new samples: Cut-and-Copy, Fetch API-ResponseStream

Impacted tests(approved): new 2, update 0, delete 0
Unit test platform: [Android][Crosswalk 14.43.341.0]
Unit test result summary: pass 2, fail 0, block 0

BUG=https://crosswalk-project.org/jira/browse/XWALK-4132

### CODE REVIEW

File: misc/sampleapp-android-tests/sampleapp/M43_AutoCapitalize.html [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+<link rel="author" title="Intel" href="http://www.intel.com">
+<div>
+  <p>Test purpose:</p>
+  <ol>
+    <li>Note: the sample need network support.</li>
+    <li>Check that text fields is supported hint autocapitalization or not.</li>
+  </ol>
+  <p>Test steps:</p>
+  <ol>
+    <li>Open the url "https://googlechrome.github.io/samples/autocapitalize/index.html"</li>
```
Comment (zqzhang) [inline misc/sampleapp-android-tests/sampleapp/M43_AutoCapitalize.html]: Use `<a>`

**Comment by zqzhang** [comment]
Hmm, I am thinking of simplifying the verification task of Chromium web platform new features enabled by Crosswalk rebasing to Chromium.

(1) In each rebase, we only need an online page, e.g. http://zqzhang.github.io/demo/chromestatus/m43.html, to list the upstream samples. And run the samples on Crosswalk Project for Android and Chrome for Android, with their results comparison reporting to crosswalk-dev mail list.

(2) For the valuable samples, such as those for features that Crosswalk Project has corresponding feature tracking in its JIRA system, we then integrate them into this sampleapp test suite.

Thoughts?

CC @haoxli @kangxu

**Comment by chunywang** [comment]
I think it is okay. we will do not need to integrate those features that crosswalk may not supports. 
CC @haoxli  @kangxu

**Comment by zqzhang** [comment]
> we will do not need to integrate those features that crosswalk may not supports

Crosswalk shall support the same features as Chromium. And that's why we do the feature verification by comparing samples running on Crosswalk and Chromium. To make sure Crosswalk supports the same features as Chromium.

I mean we don't need to integrate all the samples; we only integrate those samples which have corresponding Crosswalk JIRA feature requests.

**Comment by zqzhang** [comment]
@chunywang please update the commit message with PR description:

BUG=https://crosswalk-project.org/jira/browse/XWALK-4132

Let's check whether https://crosswalk-project.org/jira/browse/XWALK-1203 is fixed in this project, thanks.

**Comment by zqzhang** [comment]
Actually no need `[xwalk-4132]` prefix in commit message header, because there is `BUG=https://crosswalk-project.org/jira/browse/XWALK-4132` in commit message body.

**Pull request merged**